packages/base/src/widget.ts

```diff
@@ -79,9 +82,9 @@ class WidgetModel extends Backbone.Model {
             _model_name: 'WidgetModel',
             _model_module_version: JUPYTER_WIDGETS_VERSION,
             _view_module: '@jupyter-widgets/base',
-            _view_name: null,
+            _view_name: null as string,
```

Comment:
Maybe better to change the typing of this to `string | null` ? And similarly for others like this?